doc/classes/Transform3D.xml

```diff
@@ -181,8 +181,25 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
-			A transform with no translation, no rotation, and its scale being [code]1[/code]. Its [member basis] is equal to [constant Basis.IDENTITY].
-			When multiplied by another [Variant] such as [AABB] or another [Transform3D], no transformation occurs.
+			The identity [Transform3D]. This is a transform with no translation, no rotation, and not scaled. Its [member basis] is equal to [constant Basis.IDENTITY]. This also means that:
+			- Its [member Basis.x] points right ([constant Vector3.RIGHT]);
+			- Its [member Basis.y] points up ([constant Vector3.UP]);
+			- Its [member Basis.z] points back ([constant Vector3.BACK]).
+			[codeblock]
+			var transform = Transform3D.IDENTITY
+			var basis = transform.basis
+			print("| X | Y | Z | Origin")
+			print("| %s | %s | %s | %s" % [basis.x.x, basis.y.x, basis.z.x, transform.origin.x])
+			print("| %s | %s | %s | %s" % [basis.x.y, basis.y.y, basis.z.y, transform.origin.y])
+			print("| %s | %s | %s | %s" % [basis.x.z, basis.y.z, basis.z.z, transform.origin.z])
+			# Prints:
+			# | X | Y | Z | Origin
+			# | 1 | 0 | 0 | 0
+			# | 0 | 1 | 0 | 0
+			# | 0 | 0 | 1 | 0
+			[/codeblock]
+			When this constant is transformed (multiplied) by a [Vector3], an [AABB], or another [Transform3D], no transformation occurs.
+			[b]Note:[/b] In GDScript, this constant is equivalent to creating a [constructor Transform3D] without any arguments. It can be used to make your code clearer, and for consistency with C#.
```

Comment:
I am heavily considering removing "It can be used to make your code clearer, and for consistency with C#." it feels... for granted, or perhaps not the right place for recommendations(?).

Instead, I wanted to mention how this is basically the default value for Basis/Transform2D/Transform3D/Quaternion

Reply:
I think this note is useful because it explains to the user why it exists (otherwise people would ask "if it's the same as the default then why use this over the default constructor?").